tags/resources/tag-formation-templates/template-roles.md

```diff
@@ -1,3 +1,103 @@
+One of the TAG chairs serves as the working group sponsor and provides guidance to the working group leads.
+Working group leads are signed off by the TAG Chairs and the TOC Liaisons.
+The [leadership election process is described here](template-leadership-election-process.md#working-group-lead).
+
+**A working group lead is expected to**:
+* Schedule, host, plan and facilitate meetings for the working group.
+* Provide technical direction unique to the working group.
+* Plan working group deliverables.
+* Report to the TAG on the working group status.
+* Evolve the working group.
+* Garner active participation.
+* Establish documents and correct permissions for contributions to occur.
+* Contribute content to the working group.
+* Onboard and setup the working group lead successor.
+
+## Project Lead
+
+A project lead manages a community group for a specified period of time and works to achieve a specific outcome that has been previously discussed and agreed upon.
+Unlike working groups, projects require a predetermined time frame and are limited in scope and focused on a central deliverable. The project proposal must be discussed at a meeting and actively communicated to the entire TAG community.
+A TAG Chair or TL acts as a sponsor of the project and dedicates a portion of their time to actively support the effort.
+TAG Chairs must sign off on the establishment of the project group.
+Due to the limited scope of the role, there are no specific requirements for structuring the selection process for the project lead.  
+The TAG Chairs are responsible for selecting a suitable project lead and must ensure that the establishment of the project is transparent, accessible, and understood by all.
+
+**A project lead is expected to**:
+* Schedule, host, plan and facilitate meetings for the project.
+* Provide technical direction unique to the project.
+* Plan the project deliverable.
+* Report to the TAG on the project status.
+* Garner active participation.
+* Establish documents and correct permissions for contributions to occur.
+* Contribute content to the project.
+
+## TOC liaison
+
+The [CNCF TAG](https://github.com/cncf/toc/blob/master/tags) process identifies a TOC Liaison.
+The TAG Chairs are responsible for establishing effective communication with the TOC liaison, including further communication to the wider TOC upon request.
+
+The TOC Liaison will occasionally prioritize TAG activities, as needed by the TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/master/charter.md#1-mission-of-the-cloud-native-computing-foundation).
+
+## Emeritus Chair
+
+After a TAG Chair completes their term, they transition into the role of Emeritus Chair.
+This role does not impose any obligations on the TAG, but it acknowledges their committed time and effort.
+The role also lets them share information about their previous work and contributions to the TAG.
+An Emeritus Chair retains the same rights/access as in their previous role.
```

Comment:
```suggestion
An Emeritus Chair does/does not retain the same rights/access as in their previous role.
```
Some do and some don't, it depends on how the TAG chooses to operate.  For instance, now that I am Emeritus for TAG Security, i do retain permissions, but do not exercise them at this time as it would be seen as a conflict (I am no longer the liaison nor am I actively contributing to the group in a meaningful capacity as a STAG member or leader. That may change in the future. 😄 )